### PR TITLE
A11y: Add Dashboard Settings to Pa11y CI and Refactor local a11y report

### LINKED ISSUE #54010: A11y: Add Dashboard Settings to Pa11y CI
**Why we need this**:

After this [PR](https://github.com/grafana/grafana/pull/52682) is merged and to avoid future a11y regressions add Dashboard Settings pages into Pa11y CI, this means adding into [.pa11yci-pr.conf.js](https://github.com/grafana/grafana/blob/main/.pa11yci-pr.conf.js) and [.pa11yci.conf.js](https://github.com/grafana/grafana/blob/main/.pa11yci.conf.js) the URLs of dashboard settings:

- [ ] [General](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&editview=settings) 
- [ ] [Annotations](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&editview=annotations)
- [ ] [Variables](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&editview=templating)
- [ ] [Links](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&editview=links)
- [ ] [Versions](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&editview=versions)
- [ ] [Permissions](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&editview=permissions)
- [ ] [Json Model](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&editview=dashboard_json)

If you find existing a11y issues on these pages, add the corresponding threshold and verify we have a related GitHub issue to fix.

### PR DESCRIPTION
**Pull request by axelavargas**
in pa11y ci

**What this PR does / why we need it**:

- [x] Refactor scripts/generate-a11y-report.sh to use `grafana-puppeteer` image, the same image we use on the CI
- [x] Added settings pages into pa11y configuration

**Local accessibility report generated after `yarn test:accessibility-report`**

image

**Which issue(s) this PR fixes**:

Fixes #54010

**Special notes for your reviewer**:

The `yarn test:accessibility-report` does not work in M1 due to a [known issue with puppeteer,](https://github.com/puppeteer/puppeteer/issues/7916) I will create a new issue to tackle this problem in the future.

### CODE REVIEW

File: .pa11yci.conf.js [hunk 1]
```diff
@@ -1,3 +1,40 @@
+var dashboardSettings = [
```
Comment (ivanortegaalba) [inline .pa11yci.conf.js:1]: There is any reason why the scenarios like `dashboardSettings` are not shared between `.pa11yci.conf.js` and `.pa11yci-pr.conf.js`? They are the same, aren't they?

**Review by ivanortegaalba — COMMENTED** [review]
The PR looks good to me! I left a small question about duplicity. I cannot run it locally because the M1 bug, so I'll wait for someone else to test it. Thanks for this improvement!

File: .pa11yci.conf.js (hunk 1 above)
Comment (axelavargas) [reply .pa11yci.conf.js:1]: More or less the same, but the difference is that one of them has the "thresholds" property and the other doesn't (We need thresholds for the PR pipeline) :)

**Comment by axelavargas** [comment]
> The PR looks good to me! I left a small question about duplicity. I cannot run it locally because the M1 bug, so I'll wait for someone else to test it. Thanks for this improvement!

Thanks for the review 🙌🏾 , btw you can take a look at the CLI report in drone https://drone.grafana.net/grafana/grafana/77828/2/22

**Review by ivanortegaalba — APPROVED** [review]
I see the results in the CI are ✅ , and all new URLs are run  and tested.

Captura de Pantalla 2022-09-12 a las 15 07 59

**Pull request merged**